src/common/latest/esbuild.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  bundle: true,
+  platform: "node",
+  format: "cjs",
+  target: "node10",
```

Comment:
can we not target node16 to align with the latest runner? or we can do that later in its own PR

Reply:
Definitively true, I will update it!

Reply:
TY, didn't think about it, [added it to the other ticket](https://sonarsource.atlassian.net/browse/VSTS-376) for bumping the agent versions (because ATM this is still running on Node 10)